src/Huntress/Plugin/Remind.php

```diff
@@ -101,6 +103,7 @@ public static function db(Schema $schema): void
         $t->addColumn("idChannel", "bigint", ["unsigned" => true]);
         $t->addColumn("timeRemind", "datetime");
         $t->addColumn("message", "string", ['customSchemaOptions' => DatabaseFactory::CHARSET]);
+        $t->addColumn("snow", "bigint", ["unsigned" => true]);
```

Comment:
since we now have a snowflake, can we make it the primary key? `idMessage` is a snowflake too, but i'd rather we use ours if we have it.

Reply:
Done; also renamed it to `idReminder` (and $snow to $id) to make it clearer; also changed the column order to put the new primary key first.